Sources/Valkey/Commands/GeoCommands.swift

```diff
@@ -1030,15 +1073,57 @@ public struct GEOSEARCHSTORE: ValkeyCommand {
             unit.encode(into: &commandEncoder)
         }
     }
+    public struct ByPolygonVertices: RESPRenderable, Sendable, Hashable {
+        @usableFromInline let longitude: Double
+        @usableFromInline let latitude: Double
```

Comment:
I think we should make the properties public and var. This will also help with API evolution.